paymentsheet/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -222,7 +222,7 @@ internal abstract class BaseSheetViewModel(
         )
     }
 
-    private fun providePaymentMethodName(code: PaymentMethodCode?): String {
+    internal fun providePaymentMethodName(code: PaymentMethodCode?): String {
```

Comment:
Is this the right thing to use? I'd only expect this to be used for new payment methods, not saved.

Reply:
This is kinda weird. It's like making assumptions that maybe we shouldn't be. Let's pair on this later.

Reply:
https://jira.corp.stripe.com/browse/MOBILESDK-2158